person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  atrr_reader :id, :name, :age
+  atrr_writer :name, :age
```

Comment:
- [ ] I think you meant to use `attr_reader`, not `atrr_reader` (notice double 'r').

- [ ] The `:name` and `:age` variables should have getter and setter (attr_accessor), not just setter.

  In conclusion, we should have these:

  ```
  attr_reader :id
  attr_accessor :name, :age
  ```